src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.scss

```diff
@@ -1,3 +1,13 @@
 .margin-bottom-15px {
   margin-bottom: 15px;
 }
+
+// The clickable parts of the boxes should change the cursor to a pointer
+
+.tm-question-text-with-info {
+  cursor: pointer;
+}
+
+.card-header {
+  cursor: pointer;
+}
```

Comment:
Not sure why you have 2 of them. 1 should do

Reply:
Also, try not to name your classes like `.tm-question-text-with-info`. It is not descriptive